README.md

```diff
@@ -24,6 +24,21 @@
 
 Utilising [tagged template literals](https://www.styled-components.com/docs/advanced#tagged-template-literals) (a recent addition to JavaScript) and the [power of CSS](https://www.styled-components.com/docs/api#supported-css), `styled-components` allows you to write actual CSS code to style your components. It also removes the mapping between components and styles – using components as a low-level styling construct could not be easier!
 
+Utilising [Object](https://www.styled-components.com/docs/advanced#style-objects) in
+`styled-components` allows you to use object instead of string in writing your CSS code. This helps in gradually moving to styled-components from existing style objects.
+
+```JSX
+  const Button = styled.button({
+    color: 'grey'
+  })
+```
+Equivalent to:
+```JSX
+  const Button = styled.button`
+    color: 'grey'
```

Comment:
need to remove the quotes here, as it isn't valid css

Reply:
@probablyup  sorry my bad. I removed the quotes.